blog.html

```diff
@@ -39,35 +39,35 @@ <h2 class="bullet">
+                <li><b>KIP-937: Improve Message Timestamp Validation: </b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-937%3A+Improve+Message+Timestamp+Validation">KIP-937</a>. Improved Producer's record timestamp validation.</li>
+                <li><b>KIP-938: Add more metrics for measuring KRaft performance: </b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-938%3A+Add+more+metrics+for+measuring+KRaft+performance">KIP-938</a>. Added the targeted KRaft performance metrics mentioned in KIP-938 except ForwardingManager metrics.</li>
             </ul>
                 <h3>Kafka Streams</h3>
                 <ul>
-                <li><b>KIP-923: Add A Grace Period to Stream Table Join</b>: <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-923%3A+Add+A+Grace+Period+to+Stream+Table+Join">KIP-923</a>Added a grace period to Stream Table join.</li>
-                <li><b>KIP-941: Range queries to accept null lower and upper bounds</b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-941%3A+Range+queries+to+accept+null+lower+and+upper+bounds">KIP-941</a> Range queries accept lower and upper bounds as null values.</li>
+                <li><b>KIP-923: Add A Grace Period to Stream Table Join: </b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-923%3A+Add+A+Grace+Period+to+Stream+Table+Join">KIP-923</a>Added a grace period to Stream Table join.</li>
+                <li><b>KIP-941: Range queries to accept null lower and upper bounds: </b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-941%3A+Range+queries+to+accept+null+lower+and+upper+bounds">KIP-941</a> Range queries accept lower and upper bounds as null values.</li>
                 </ul>
                 <h3>Kafka Connect</h3>
                 <ul>
-                    <li><b>KIP-793: Allow sink connectors to be used with topic-mutating SMTs</b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-793%3A+Allow+sink+connectors+to+be+used+with+topic-mutating+SMTs">KIP-793</a>. It supports topic-mutating SMTs for async sink connectors.</li>
-                    <li><b>KIP-875: First-class offsets support in Kafka Connect</b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-875%3A+First-class+offsets+support+in+Kafka+Connect">KIP-875</a>. Kafka Connect supports offset reset.</li>
-                    <li><b>KIP-898: Modernize Connect plugin discovery</b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-898%3A+Modernize+Connect+plugin+discovery">KIP-898</a>.The worker will read plugin classes from ServiceLoader manifests and module info during startup.</li>
+                    <li><b>KIP-793: Allow sink connectors to be used with topic-mutating SMTs: </b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-793%3A+Allow+sink+connectors+to+be+used+with+topic-mutating+SMTs">KIP-793</a>. It supports topic-mutating SMTs for async sink connectors.</li>
```

Comment:
What about: `This allows transformations that update the topic of records to be used with sink connectors that use their own mechanism to track offsets.`?